support/web/css/vars.scss

```
@@ -105,3 +105,6 @@ $light-button-hover-bg: $zinc-200;
 
 $dark-text-bg: #23272E;
 $dark-text-fg: #ABB2BF;
+
+// "Desktop layout": navigation bar, content, sidenotes.
+$desktop-layout-width: 110rem;
```

Given https://github.com/plt-amy/1lab/issues/147, we should keep this low. The minimum for my 1920x1080, 120 DPI laptop screen seems to be 1920*96/120/16 = 96rem, minus one for safety:

```suggestion
$desktop-layout-width: 95rem;
```